src/_includes/layouts/contact.liquid

```diff
@@ -1,31 +1,19 @@
 ---
 layout: layouts/base.liquid
 ---
-<div class="bg-parallax-photo">
-  <div
-    :speed-factor="0.5"
-    sectionClass="parallax-outside"
-    containerClass="parallax-inside">
-    <img src="/assets/img/bg-triangle-blue2.png" class="float-right float-md-none" />
+<div class="flex flex-col sm:flex-row sm:justify-center md:justify-around">
+  <div class="py-5 px-4">
+    <h1 class="absolute text-blue px-4 md:mt-12 md:mb-12">{{ contactHeader }}</h1>
+     <img src="/assets/img/bg-triangle-blue2.png" alt="Blue triangle"/>
```

Comment:
I just viewed the image for this, probably want to make the alt text more descriptive.  Something like, "Image of girl shaking hands."